nim/app.py

```diff
@@ -34,12 +39,16 @@ def update_game_state():
 
 @app.route('/new', methods=['GET', 'POST'])
 def new_game():
-    json = loads('{"min": 3, "max": 12, "piles": 3}')
-    state = app.game.new_game(
-        min=json['min'],
-        max=json['max'],
-        piles=json['piles'],
-    )
+    """
+    route for /new so user can start new game
+    Uses JSON from frontend for params in new game
+    """
+    user_request = request.get_json()
+    game_request = {**DEFAULT_GAME, **user_request}
```

Comment:
That shouldn't be needed. If some values aren't passed when calling `new_game()` , that function should have the right defaults (i.e. you need to modify `new_game()` so it has default values for its arguments).